roles/ceph-osd/templates/ceph-osd-run.sh.j2

```diff
@@ -44,8 +44,8 @@ create_dev_list $1
   -v /etc/localtime:/etc/localtime:ro \
   --device=/dev/${1} \
   --device=/dev/${1}1 \
-  {% if raw_journal_devices|length > 0 -%}
-  -e OSD_JOURNAL={{ raw_journal_devices[0] }} \
+  {% if raw_journal_devices -%}
+  -e OSD_JOURNAL={{ raw_journal_devices|random }} \
```

Comment:
Note for myself, we need a random but predictable value here. We don't want to get a new value each time we run Ansible...